doc-tools/missing-doclet/src/main/java/org/opensearch/missingdoclet/MissingDoclet.java

```diff
@@ -59,6 +60,7 @@ public class MissingDoclet extends StandardDoclet {
     private static final int METHOD = 2;
     // checks that @param tags are present for any method/constructor parameters
     private static final int PARAMETER = 3;
+    // TODO Should we stick this list in missing-javadoc.gradle instead?
```

Comment:
Seems like out of date, right?

Reply:
Thanks for pointing it out.  It'll be removed.